tests/testthat/test-dfm.R

```diff
@@ -8,10 +8,10 @@ test_that("oldest dfm test", {
                               taxation="taxation",
                               taxregex="tax*",
                               country="united_states"))
-    dictDfm <- dfm(mycorpus, dictionary = mydict, valuetype = "glob")
-    dictDfm[1:10, ]
-    thesDfm <- dfm(mycorpus, thesaurus = mydict, valuetype = "glob")
-    thesDfm[1:10, (nfeature(thesDfm)-8) : nfeature(thesDfm)]
+    dictDfm <- dictDfm <- dfm(mycorpus, dictionary = mydict, valuetype = "glob")
+    dictDfm <- dictDfm[1:10, ]
+    dictDfm <- thesDfm <- dfm(mycorpus, thesaurus = mydict, valuetype = "glob")
```

Comment:
I'm not sure what the point of these assignments are, since they keep overwriting `dictDfm`. Shouldn't there be some `expect_*()` calls around these to make them tests?